packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/sync/SyncConfiguration.kt

```diff
@@ -375,12 +423,13 @@ public interface SyncConfiguration : Configuration {
 
         override fun build(): SyncConfiguration {
             val allLoggers = userLoggers.toMutableList()
+            val defaultSystemLogger: RealmLogger = createDefaultSystemLogger(Realm.DEFAULT_LOG_TAG)
```

Comment:
Because we need to use it in the default strategies for logging messages. Before it was only defined locally. I could have grabbed it by doing `allLoggers[0]` but I think it's more visible this way.

Reply:
Reverted back to use app logger. Using the system logger didn't allow any custom logger to receive client reset messages.